monkey_island/cc/services/config.py

```diff
@@ -17,60 +20,60 @@
             "type": "string",
```

Comment:
Can we move this entire schema to a seperate file? Out of 970 lines, 785 are the schema, making the file hard to read.

Reply:
Yeah. But let's not do it in this branch. I've added as task to tasklist